spec/api-desktop-capturer-spec.ts

```diff
@@ -99,7 +99,7 @@ ifdescribe(!process.arch.includes('arm') && process.platform !== 'win32')('deskt
     expect(isEmpties.every(e => e === true)).to.be.true();
   });
 
-  it('getMediaSourceId should match DesktopCapturerSource.id', async () => {
+  it('getMediaSourceId should match DesktopCapturerSource.id', async function () {
```

Comment:
Note: using `function() {}` instead of `() => {}` here and below so that the `this` capture works for `this.skip()`. In fact, Mocha [recommends against arrow functions for this reason](https://mochajs.org/#arrow-functions)